apps/tangent-electron/src/app/model/commands/CreateNewFile.ts

```diff
@@ -248,7 +248,16 @@ export default class CreateNewFileCommand extends WorkspaceCommand {
 
 				this.workspace.navigateTo(nav)
 			}
-	
+            if (newNode instanceof File && context?.rule) {
+                const templateValue = typeof context.rule.template === 'string' ?
```

Comment:
Doesn't the template need to have its replacement variables injected here?

Also, this block is using spaces for indentation instead of tabs.